tokens/manage/multisig-safe.md

```diff
@@ -312,4 +313,36 @@ To double check that the correct text was set, you can go through the process ag
 
 Congratulations, you've successfully interacted with a smart contract using the Safe!
 
+## Using Moonbeam Safe APIs
+There are APIs available to read from and interact with Moonbeam Safes for Moonbase Alpha, Moonriver, and Moonbeam.
+
+|     Network    |                                           Multisig API Page                                              |
+|:--------------:|:--------------------------------------------------------------------------------------------------------:|
+|    Moonbeam    |  [{{ networks.moonbeam.multisig.api_page }}]({{ networks.moonbeam.multisig.api_page }}){target=_blank}   |
+|    Moonriver   | [{{ networks.moonriver.multisig.api_page }}]({{ networks.moonriver.multisig.api_page }}){target=_blank}  |
+| Moonbase Alpha |  [{{ networks.moonbase.multisig.api_page }}]({{ networks.moonbase.multisig.api_page }}){target=_blank}   |
+
+### Retrieve Safe Information
+You can retrieve information about Safes from the Moonbeam Safe API. From the Safe page, copy the address of your Safe:
+
+![Contract Interaction Call Result](/images/tokens/manage/multisig-safe/safe-30.png)
```

Comment:
for this image, there's actually a copy icon you can click on the left hand side panel i think pointing to that might be easier and then you don't have to write "copy" next to the arrow

Reply:
The reason I told them to copy it was because the button copies with "mbase", so it would turn out "mbase:0x..."

We can try changing this in the actual website?

Reply:
Ah I see... I really just wasn't a big fan of "copy" next to the arrow so I just removed it. The copy button actually does the same thing in the Rinkeby implementation where it has `rin:0x...` so I don't think we should change it here